c/algorithm/BubbleSort.c

```diff
@@ -0,0 +1,11 @@
+static void BubbleSort(int[] arr){
+	for (int i = 0; i < arr.Length - 1; i++)	{
```

Comment:
`arr.Length` does not exists in c, you have to use `n` instead.
Same for the next line